packages/react-components/react-tabster/package.json

```diff
@@ -37,7 +37,7 @@
     "@griffel/react": "^1.5.14",
     "@swc/helpers": "^0.5.1",
     "keyborg": "^2.5.0",
-    "tabster": "^6.0.1"
+    "tabster": "^7.0.1"
```

Comment:
@mshoho does this break the Types api you presented on tech sync ? What's the status of those changes ?

is this safe to bump ? any other behind the scenes breaking changes that can affect users ?

Reply:
Every bundle size increase is driven by adding new features or fixing bugs. I am not sure what you mean by «mitigate» in this context. I am also not sure why you call these numbers alarming. It is gradual development. It grows proportionally to the amount of covered use cases. And there is only one Tabster in the bundle, you cannot sum the bundle size increases from the bundle size reports. It's not 563 bytes per component. It's 563 bytes per entire application (even if you use all components at once).

Reply:
The PR that addresses the Types: https://github.com/microsoft/fluentui/pull/30969
This one should be closed in favour of that one.